lib/client.ts

```diff
@@ -36,13 +36,18 @@ function authValid(response): any {
 }
 
 function request(url: string, payload: any, method: string): Function {
+  const mode: RequestMode = 'cors';
+  const credentials: RequestCredentials = 'include';
   const headers = new Headers()
   headers.append('Accept', 'application/json')
+  headers.append('Content-Type', 'application/json');
 
   return function authorizedRequest([authKey, authValue]: Array<string>): Promise<SDKResponse> {
     headers.append(authKey, authValue)
 
-    const request = fetch(url, { mode: 'cors', credentials: 'include', method, headers });
+    const defaultOpts = { mode, credentials, method, headers };
+    const opts = payload ? Object.assign(defaultOpts, { body: JSON.stringify(payload) }) : defaultOpts;
```

Comment:
I thought about defaulting the param to an empty object and sending it all the time, but I think its probably safer to not include it on non-body type requests. https://stackoverflow.com/a/983458/1512732